frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/menu-actions.ts

```diff
@@ -0,0 +1,18 @@
+import { K8sKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { asAccessReview, Kebab } from '@console/internal/components/utils';
+import { attachDeploymentToOBCModal } from './modals/attch-deployment-obc-modal';
+
+const attachDeployment = (kind: K8sKind, resource: K8sResourceKind) => ({
+  label: `Attach to Deployment`,
```

Comment:
Why there is need of using string literal? You can directly use quotes.

Reply:
ack